assets/airports/lipz.json

```diff
@@ -0,0 +1,2300 @@
+{
+    "airac": 2002,
+    "radio": {
+        "twr": "Venezia Tower",
+        "app": "Venezia Radar",
+        "dep": "Venezia Radar"
+    },
+    "icao": "LIPZ",
+    "iata": "VCE",
+    "magnetic_north": 3.0,
+    "ctr_radius": 90,
+    "ctr_ceiling": 12500,
+    "initial_alt": 4000,
+    "position": ["N45d30m19.0", "E012d21m07.0", "7ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": ["N45d30m19.0", "E012d21m07.0"],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 50,
+        "speed": 6
+    },
+    "arrivalRunway": "04R",
+    "departureRunway": "04R",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
+        {
+            "floor": 20,
+            "ceiling": 125,
+            "airspace_class": "D",
```

Comment:
Airspace class should be the American equivalent, which is probably B. (Feel free to prove me wrong)